frontend/src/components/admin/ApprovedLanguagesTable.tsx

```diff
@@ -37,9 +37,20 @@ type ApprovedLanguage = {
   sliderValue: number;
 };
 
+type ApprovedLanguagesTableProps = {
```

Comment:
Moved the `ApprovedLanguagesTableProps` so that it would be right above `const ApprovedLanguagesTable = ...`